src/tests/ftest/util/dmg_utils_params.py

```diff
@@ -22,17 +22,22 @@
   portions thereof marked with this legend must also reproduce the markings.
 """
 
+#from command_utils_base import \
+#    BasicParameter, YamlParameters, TransportCredentials
 from command_utils_base import \
-    BasicParameter, YamlParameters, TransportCredentials
+    BasicParameter, LogParameter, YamlParameters, TransportCredentials, \
+    EnvironmentVariables
```

Comment:
The `EnvironmentVariables` class is not used in these changes:
```suggestion
    BasicParameter, LogParameter, TransportCredentials, YamlParameters
```